src/lib/core/datetime/date-adapter.ts

```diff
@@ -192,6 +187,26 @@ export abstract class DateAdapter<D> {
   abstract isValid(date: D): boolean;
 
   /**
+   * Attempts to deserialize a value to a valid date object. This is different from parsing in that
+   * deserialize should only accept non-ambiguous, locale-independent values (e.g. a ISO 8601
+   * string). The default implementation does not allow any deserialization, it simply checks that
+   * the given value is already a valid date object or null. The `<mat-datepicker>` will call this
+   * method on all of it's `@Input()` properties that accept dates. It is therefore possible to
+   * support passing your wire format directly to these properties by overriding this method to
+   * also deserialize your wire format.
```

Comment:
Say something like "This will generally be used for data coming directory from some backend or database." ?

Reply:
Isn't this just saying the same thing as the "wire format" part above?

Reply:
Yeah, but it's a bit more explicit (less jargony?) than "wire format"

Reply:
Ok, re-worded it to say backend